content/en/docs/tutorials/cluster-management/install-use-dra.md

```diff
@@ -0,0 +1,540 @@
+experience should be like) and also potentially goes into much more detailed/ ✨
+fun ✨ use cases. 
+-->
+
+{{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
+
+
+<!-- overview -->
+This tutorial shows you how to install {{< glossary_tooltip term_id="dra"
+text="DRA" >}} drivers in your cluster and how to use them in conjunction with
+the DRA APIs to allocate {{< glossary_tooltip text="devices" term_id="device"
+>}} to Pods. This page is intended for cluster administrators.
+
+{{< glossary_tooltip text="Dynamic Resource Allocation (DRA)" term_id="dra" >}}
+is a Kubernetes feature that allows a cluster to manage availability and
+allocation of hardware resources to satisfy Pod-based claims for hardware
+requirements and preferences (see the [DRA Concept
+page](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) for more
+background). To support this, a mixture of Kubernetes built-in components (like
+the Kubernetes scheduler, kubelet, and kube-controller-manager) and third-party
+components (called DRA drivers) share the responsibility to advertise, allocate,
+prepare, mount, healthcheck, unprepare, and cleanup resources throughout the Pod
+lifecycle. These components share information via a series of DRA specific APIs
+in the `resource.k8s.io` API group, including {{< glossary_tooltip
+text="DeviceClasses" term_id="deviceclass" >}}, {{< glossary_tooltip
+text="ResourceSlices" term_id="resourceslice" >}}, {{< glossary_tooltip
+text="ResourceClaims" term_id="resourceclaim" >}}, as well as new fields in the
+Pod spec itself.
+
+<!-- objectives -->
+
+### {{% heading "objectives" %}}
+* Deploy an example DRA driver
+* Deploy a Pod requesting a hardware claim using DRA APIs
+* Delete a Pod that has a claim
+
+<!-- prerequisites -->
+## {{% heading "prerequisites" %}}
+
+Your cluster should support [RBAC](/docs/reference/access-authn-authz/rbac/).
+You can try this tutorial with a cluster using
+a different authorization mechanism, but in that case you will have to adapt the
+steps around defining roles and permissions.
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+This tutorial has been tested with Linux nodes, though it may also work with
+other types of nodes.
+
+ {{< version-check >}}
+
+Your cluster also must be configured to use the Dynamic Resource Allocation
+feature.
+To enable the DRA feature, you must enable the following feature gates and API groups:
+
+1.  Enable the `DynamicResourceAllocation`
+    [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+    on all of the following components:
+   
+    * `kube-apiserver`
+    * `kube-controller-manager`
+    * `kube-scheduler`
+    * `kubelet`
+
+1.  Enable the following
+    {{< glossary_tooltip text="API groups" term_id="api-group" >}}:
+
+    * `resource.k8s.io/v1beta1`: required for DRA to function.
+    * `resource.k8s.io/v1beta2`: optional, recommended improvements to the user
+      experience.
```

Comment:
Can we mention something like how v1beta2 isn't required for DRA to work in general, but it is required for the steps in the tutorial since it involves creating a ResourceClaim defined in that version? Or if that muddies this too much, only say to enable both v1beta1 and v1beta2 without offering details as to why each is required? Or redefine the ResourceClaim in v1beta1?